### DISCUSSION TITLE
Could we enable Github Copilot in the Finder view in PHPStorm?

### DISCUSSION THREAD

**Discussion by Cuji12** [community/community#65466] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

It'd be ideal to not have to open the file explicitly to get the advantages of Github Copilot, currently it doesn't run in the Finder window - it'd be great if it would!

**Comment by Akash1134** [comment]
Hi @Cuji12 

Thank you for sharing the feedback. 

As a GitHub user, the most you can do is to share your feedback on this Discussion space with the correct tags (include examples), and trust that we are always working to make GitHub as awesome as possible. We can't guarantee that every bit of feedback sees action, but we are paying attention.
